src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -433,6 +433,18 @@ public static class Parser
         private static readonly Regex AnimeReleaseGroupRegex = new Regex(@"^(?:\[(?<subgroup>(?!\s).+?(?<!\s))\](?:_|-|\s|\.)?)",
                                                                 RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
+        //Handle Exception Release Groups that don't follow -RlsGrp; Manual List
+        //groups whose releases end with RlsGroup) or RlsGroup]
+        /* Current list from Radarr 2021-08-11 PR#6494
+           Commenting out until seen in the wild; but implementing with ExceptionReleaseGroupRegexExact
```

Comment:
Just remove it. These aren't gonna apply to Sonarr anyway.

Reply:
rmvd